nanoFramework.Tools.DebugLibrary.Shared/PortSerial/PortSerialManager.cs

```diff
@@ -611,7 +611,7 @@ private bool CheckValidNanoFrameworkSerialDevice(NanoDevice<NanoSerialDevice> de
                     }
                 }
             }
-            catch
+            catch (Exception /* ex */)   // we could eat simple programming errors here - like a bad cast or other problem when changing code
```

Comment:
Your comment is correct, of course! 
This was (is) here because of the bad things always happening when we where using the UWP APIs. 
Probably needs to be revisited now that we aren't  using that anymore.

I'm no big fan of "catch all" like this one, but sometimes that's the way to go...